Cargo.lock

```diff
@@ -4796,4 +4796,4 @@ checksum = "9fd07cbbc53846d9145dbffdf6dd09a7a0aa52be46741825f5c97bdd4f73f12b"
 dependencies = [
  "cc",
  "libc",
-]
+]
```

Comment:
Do not attempt to resolve conflicts manually, simply reset file and run cargo build.

Reply:
How do I resolve it now. When I try to rebase with the develop branch(or reset it), it doesn't show me any changes for `Cargo.lock` and hence not able to commit it.